docs/conf.py

```diff
@@ -205,6 +205,7 @@
     "includehidden": False,
 }
 htmlhelp_basename = "PlasmaPydoc"
+github_url = "https://github.com/PlasmaPy/PlasmaPy"
```

Comment:
Need to verify that the "Edit on GitHub" button on the docs is working. 

There's chance that `github_url` be in the `html_theme_options` dict above instead of as a separate variable.